src/Core/src/Platform/iOS/ContainerViewController.cs

```diff
@@ -84,9 +84,9 @@ protected virtual UIView CreatePlatformView(IElement view)
 		public override void ViewDidLayoutSubviews()
 		{
 			base.ViewDidLayoutSubviews();
-			if (currentPlatformView == null)
+			if (currentPlatformView == null || !IsViewLoaded || View == null)
 				return;
-			currentPlatformView.Frame = View!.Bounds;
+			currentPlatformView.Frame = View.Bounds;
```

Comment:
This fixes the above crash when running these tests in Debug mode on a simulator ^^

But now a new one:

```
Unhandled Exception:
  ObjCRuntime.ObjCException: Objective-C exception thrown.  Name: NSInternalInconsistencyException Reason: <__MonoMac_NSAsyncActionDispatcher: 0x60000228b560>: An -observeValueForKeyPath:ofObject:change:context: message was received but not handled.
  Key path: bounds
  Observed object: <CALayer: 0x60000228b0e0>
  Change: {
      kind = 1;
      new = "NSRect: {{0, 0}, {1024, 1366}}";
      old = "NSRect: {{0, 0}, {0, 0}}";
  }
  Context: 0x60000228b560
  Native stack trace:
  	0   CoreFoundation                      0x000000010c6288ab __exceptionPreprocess + 242
  	1   libobjc.A.dylib                     0x000000011daacba3 objc_exception_throw + 48
  	2   Foundation                          0x0000000113eb39b9 -[NSObject(NSKeyValueObserverRegistration) _addObserver:forProperty:options:context:] + 0
  	3   Foundation                          0x0000000113eb3e93 NSKeyValueNotifyObserver + 304
  	4   Foundation                          0x0000000113eb76f0 NSKeyValueDidChange + 462
  	5   Foundation                          0x0000000113eb338c NSKeyValueDidChangeWithPerThreadPendingNotifications + 146
  	6   QuartzCore                          0x00000001176f697d _ZN2CA5Layer10end_changeEPNS_11TransactionEjP11objc_objectS4_ + 53
  	7   QuartzCore                          0x00000001176ecaf3 -[CALayer setBounds:] + 511
  	8   QuartzCore                          0x00000001176ed885 -[CALayer setFrame:] + 595
  	9   UIKitCore                           0x0000000128d96b50 -[UIView _backing_setFrame:] + 83
  	10  UIKitCore                           0x00000001299a9cb3 -[UIView(Geometry) setFrame:] + 356
  	11  libxamarin-dotnet-debug.dylib       0x0000000109f1149c xamarin_dyn_objc_msgSendSuper + 220
  	12  ???                                 0x000000015c83eca7 0x0 + 5847116967
  	13  libmonosgen-2.0.dylib               0x000000010a6d842a ves_pinvoke_method + 474
  	14  libmonosgen-2.0.dylib               0x000000010a6ca6a5 interp_exec_method + 3717
  	15  libmonosgen-2.0.dylib               0x000000010a6c7e53 interp_runtime_invoke + 259
  	16  libmonosgen-2.0.dylib               0x000000010a4ee89d mono_runtime_try_invoke + 157
  	17  libmonosgen-2.0.dylib               0x000000010a4f0a2f mono_runtime_invoke + 95
  	18  libxamarin-dotnet-debug.dylib       0x0000000109f086b0 xamarin_invoke_trampoline + 6320
  	19  libxamarin-dotnet-debug.dylib       0x0000000109f10019 xamarin_arch_trampoline + 105
  	20  libxamarin-dotnet-debug.dylib       0x0000000109f111fe xamarin_x86_64_common_trampoline + 118
  	21  UIKitCore                           0x00000001299c87b3 -[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 2837
  	22  QuartzCore                          0x00000001176fb02c _ZN2CA5Layer16layout_if_neededEPNS_11TransactionE + 526
  	23  QuartzCore                          0x00000001177065b9 _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 65
  	24  QuartzCore                          0x0000000117617b0c _ZN2CA7Context18commit_transactionEPNS_11TransactionEdPd + 626
  	25  QuartzCore                          0x000000011764f69b _ZN2CA11Transaction6commitEv + 735
  	26  UIKitCore                           0x00000001293ca35b _afterCACommitHandler + 105
  	27  CoreFoundation                      0x000000010c587055 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
  	28  CoreFoundation                      0x000000010c5819c2 __CFRunLoopDoObservers + 515
  	29  CoreFoundation                      0x000000010c581f0d __CFRunLoopRun + 1161
  	30  CoreFoundation                      0x000000010c5816a7 CFRunLoopRunSpecific + 560
  	31  GraphicsServices                    0x00000001217e928a GSEventRunModal + 139
  	32  UIKitCore                           0x0000000129395ad3 -[UIApplication _run] + 994
  	33  UIKitCore                           0x000000012939a9ef UIApplicationMain + 123
  	34  libxamarin-dotnet-debug.dylib       0x0000000109ecba5a xamarin_UIApplicationMain + 58
  	35  libmonosgen-2.0.dylib               0x000000010a6d95f9 do_icall + 345
  	36  libmonosgen-2.0.dylib               0x000000010a6d8103 do_icall_wrapper + 291
  	37  libmonosgen-2.0.dylib               0x000000010a6ca4ff interp_exec_method + 3295
  	38  libmonosgen-2.0.dylib               0x000000010a6c7e53 interp_runtime_invoke + 259
  	39  libmonosgen-2.0.dylib               0x000000010a4ecde8 mono_runtime_invoke_checked + 136
  	40  libmonosgen-2.0.dylib               0x000000010a4f4dfc do_exec_main_checked + 92
  	41  libmonosgen-2.0.dylib               0x000000010a623e42 mono_jit_exec + 354
  	42  libxamarin-dotnet-debug.dylib       0x0000000109f0fe77 xamarin_main + 1927
  	43  Microsoft.Maui.Controls.DeviceTests 0x0000000100f50e14 main + 68
  	44  dyld                                0x0000000109ba92bf start_sim + 10
  	45  ???                                 0x00000002016aa52e 0x0 + 8613700910
  	46  ???                                 0x0000000000000003 0x0 + 3
  
     at ObjCRuntime.Runtime.ThrowException(IntPtr gchandle)
     at UIKit.UIApplication.UIApplicationMain(Int32 argc, String[] argv, IntPtr principalClassName, IntPtr delegateClassName)
     at UIKit.UIApplication.Main(String[] args, Type principalClass, Type delegateClass)
     at Microsoft.Maui.DeviceTests.Program.Main(String[] args) in /Users/jopepper/src/maui/src/Controls/tests/DeviceTests/Microsoft.Maui.TestUtils.DeviceTests.Runners.SourceGen/Microsoft.Maui.TestUtils.DeviceTests.Runners.SourceGen.CodeBehindGenerator/TestRunner.iOS.sg.cs:line 18
  Native stack trace:
  	0   CoreFoundation                      0x000000010c6288ab __exceptionPreprocess + 242
  	1   libobjc.A.dylib                     0x000000011daacba3 objc_exception_throw + 48
  	2   Foundation                          0x0000000113eb39b9 -[NSObject(NSKeyValueObserverRegistration) _addObserver:forProperty:options:context:] + 0
  	3   Foundation                          0x0000000113eb3e93 NSKeyValueNotifyObserver + 304
  	4   Foundation                          0x0000000113eb76f0 NSKeyValueDidChange + 462
  	5   Foundation                          0x0000000113eb338c NSKeyValueDidChangeWithPerThreadPendingNotifications + 146
  	6   QuartzCore                          0x00000001176f697d _ZN2CA5Layer10end_changeEPNS_11TransactionEjP11objc_objectS4_ + 53
  	7   QuartzCore                          0x00000001176ecaf3 -[CALayer setBounds:] + 511
  	8   QuartzCore                          0x00000001176ed885 -[CALayer setFrame:] + 595
  	9   UIKitCore                           0x0000000128d96b50 -[UIView _backing_setFrame:] + 83
  	10  UIKitCore                           0x00000001299a9cb3 -[UIView(Geometry) setFrame:] + 356
  	11  libxamarin-dotnet-debug.dylib       0x0000000109f1149c xamarin_dyn_objc_msgSendSuper + 220
  	12  ???                                 0x000000015c83eca7 0x0 + 5847116967
  	13  libmonosgen-2.0.dylib               0x000000010a6d842a ves_pinvoke_method + 474
  	14  libmonosgen-2.0.dylib               0x000000010a6ca6a5 interp_exec_method + 3717
  	15  libmonosgen-2.0.dylib               0x000000010a6c7e53 interp_runtime_invoke + 259
  	16  libmonosgen-2.0.dylib               0x000000010a4ee89d mono_runtime_try_invoke + 157
  	17  libmonosgen-2.0.dylib               0x000000010a4f0a2f mono_runtime_invoke + 95
  	18  libxamarin-dotnet-debug.dylib       0x0000000109f086b0 xamarin_invoke_trampoline + 6320
  	19  libxamarin-dotnet-debug.dylib       0x0000000109f10019 xamarin_arch_trampoline + 105
  	20  libxamarin-dotnet-debug.dylib       0x0000000109f111fe xamarin_x86_64_common_trampoline + 118
  	21  UIKitCore                           0x00000001299c87b3 -[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 2837
  	22  QuartzCore                          0x00000001176fb02c _ZN2CA5Layer16layout_if_neededEPNS_11TransactionE + 526
  	23  QuartzCore                          0x00000001177065b9 _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 65
  	24  QuartzCore                          0x0000000117617b0c _ZN2CA7Context18commit_transactionEPNS_11TransactionEdPd + 626
  	25  QuartzCore                          0x000000011764f69b _ZN2CA11Transaction6commitEv + 735
  	26  UIKitCore                           0x00000001293ca35b _afterCACommitHandler + 105
  	27  CoreFoundation                      0x000000010c587055 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
  	28  CoreFoundation                      0x000000010c5819c2 __CFRunLoopDoObservers + 515
  	29  CoreFoundation                      0x000000010c581f0d __CFRunLoopRun + 1161
  	30  CoreFoundation                      0x000000010c5816a7 CFRunLoopRunSpecific + 560
  	31  GraphicsServices                    0x00000001217e928a GSEventRunModal + 139
  	32  UIKitCore                           0x0000000129395ad3 -[UIApplication _run] + 994
  	33  UIKitCore                           0x000000012939a9ef UIApplicationMain + 123
  	34  libxamarin-dotnet-debug.dylib       0x0000000109ecba5a xamarin_UIApplicationMain + 58
  	35  libmonosgen-2.0.dylib               0x000000010a6d95f9 do_icall + 345
  	36  libmonosgen-2.0.dylib               0x000000010a6d8103 do_icall_wrapper + 291
  	37  libmonosgen-2.0.dylib               0x000000010a6ca4ff interp_exec_method + 3295
  	38  libmonosgen-2.0.dylib               0x000000010a6c7e53 interp_runtime_invoke + 259
  	39  libmonosgen-2.0.dylib               0x000000010a4ecde8 mono_runtime_invoke_checked + 136
  	40  libmonosgen-2.0.dylib               0x000000010a4f4dfc do_exec_main_checked + 92
  	41  libmonosgen-2.0.dylib               0x000000010a623e42 mono_jit_exec + 354
  	42  libxamarin-dotnet-debug.dylib       0x0000000109f0fe77 xamarin_main + 1927
  	43  Microsoft.Maui.Controls.DeviceTests 0x0000000100f50e14 main + 68
  	44  dyld                                0x0000000109ba92bf start_sim + 10
  	45  ???                                 0x00000002016aa52e 0x0 + 8613700910
  	46  ???                                 0x0000000000000003 0x0 + 3
  2023-04-26 16:58:05.862517-0500 Microsoft.Maui.Controls.DeviceTests[83750:3729378] Unhandled managed exception: Objective-C exception thrown.  Name: NSInternalInconsistencyException Reason: <__MonoMac_NSAsyncActionDispatcher: 0x60000228b560>: An -observeValueForKeyPath:ofObject:change:context: message was received but not handled.
  Key path: bounds
  Observed object: <CALayer: 0x60000228b0e0>
  Change: {
      kind = 1;
      new = "NSRect: {{0, 0}, {1024, 1366}}";
      old = "NSRect: {{0, 0}, {0, 0}}";
  }
  Context: 0x60000228b560
  Native stack trace:
  	0   CoreFoundation                      0x000000010c6288ab __exceptionPreprocess + 242
  	1   libobjc.A.dylib                     0x000000011daacba3 objc_exception_throw + 48
  	2   Foundation                          0x0000000113eb39b9 -[NSObject(NSKeyValueObserverRegistration) _addObserver:forProperty:options:context:] + 0
  	3   Foundation                          0x0000000113eb3e93 NSKeyValueNotifyObserver + 304
  	4   Foundation                          0x0000000113eb76f0 NSKeyValueDidChange + 462
  	5   Foundation                          0x0000000113eb338c NSKeyValueDidChangeWithPerThreadPendingNotifications + 146
  	6   QuartzCore                          0x00000001176f697d _ZN2CA5Layer10end_changeEPNS_11TransactionEjP11objc_objectS4_ + 53
  	7   QuartzCore                          0x00000001176ecaf3 -[CALayer setBounds:] + 511
  	8   QuartzCore                          0x00000001176ed885 -[CALayer setFrame:] + 595
  	9   UIKitCore                           0x0000000128d96b50 -[UIView _backing_setFrame:] + 83
  	10  UIKitCore                           0x00000001299a9cb3 -[UIView(Geometry) setFrame:] + 356
  	11  libxamarin-dotnet-debug.dylib       0x0000000109f1149c xamarin_dyn_objc_msgSendSuper + 220
  	12  ???                                 0x000000015c83eca7 0x0 + 5847116967
  	13  libmonosgen-2.0.dylib               0x000000010a6d842a ves_pinvoke_method + 474
  	14  libmonosgen-2.0.dylib               0x000000010a6ca6a5 interp_exec_method + 3717
  	15  libmonosgen-2.0.dylib               0x000000010a6c7e53 interp_runtime_invoke + 259
  	16  libmonosgen-2.0.dylib               0x000000010a4ee89d mono_runtime_try_invoke + 157
  	17  libmonosgen-2.0.dylib               0x000000010a4f0a2f mono_runtime_invoke + 95
  	18  libxamarin-dotnet-debug.dylib       0x0000000109f086b0 xamarin_invoke_trampoline + 6320
  	19  libxamarin-dotnet-debug.dylib       0x0000000109f10019 xamarin_arch_trampoline + 105
  	20  libxamarin-dotnet-debug.dylib       0x0000000109f111fe xamarin_x86_64_common_trampoline + 118
  	21  UIKitCore                           0x00000001299c87b3 -[UIView(CALayerDelegate) layoutSublayersOfLayer:] + 2837
  	22  QuartzCore                          0x00000001176fb02c _ZN2CA5Layer16layout_if_neededEPNS_11TransactionE + 526
  	23  QuartzCore                          0x00000001177065b9 _ZN2CA5Layer28layout_and_display_if_neededEPNS_11TransactionE + 65
  	24  QuartzCore                          0x0000000117617b0c _ZN2CA7Context18commit_transactionEPNS_11TransactionEdPd + 626
  	25  QuartzCore                          0x000000011764f69b _ZN2CA11Transaction6commitEv + 735
  	26  UIKitCore                           0x00000001293ca35b _afterCACommitHandler + 105
  	27  CoreFoundation                      0x000000010c587055 __CFRUNLOOP_IS_CALLING_OUT_TO_AN_OBSERVER_CALLBACK_FUNCTION__ + 23
  	28  CoreFoundation                      0x000000010c5819c2 __CFRunLoopDoObservers + 515
  	29  CoreFoundation                      0x000000010c581f0d __CFRunLoopRun + 1161
  	30  CoreFoundation                      0x000000010c5816a7 CFRunLoopRunSpecific + 560
  	31  GraphicsServices                    0x00000001217e928a GSEventRunModal + 139
  	32  UIKitCore                           0x0000000129395ad3 -[UIApplication _run] + 994
  	33  UIKitCore                           0x000000012939a9ef UIApplicationMain + 123
  	34  libxamarin-dotnet-debug.dylib       0x0000000109ecba5a xamarin_UIApplicationMain + 58
  	35  libmonosgen-2.0.dylib               0x000000010a6d95f9 do_icall + 345
  	36  libmonosgen-2.0.dylib               0x000000010a6d8103 do_icall_wrapper + 291
  	37  libmonosgen-2.0.dylib               0x000000010a6ca4ff interp_exec_method + 3295
  	38  libmonosgen-2.0.dylib               0x000000010a6c7e53 interp_runtime_invoke + 259
  	39  libmonosgen-2.0.dylib               0x000000010a4ecde8 mono_runtime_invoke_checked + 136
  	40  libmonosgen-2.0.dylib               0x000000010a4f4dfc do_exec_main_checked + 92
  	41  libmonosgen-2.0.dylib               0x000000010a623e42 mono_jit_exec + 354
  	42  libxamarin-dotnet-debug.dylib       0x0000000109f0fe77 xamarin_main + 1927
  	43  Microsoft.Maui.Controls.DeviceTests 0x0000000100f50e14 main + 68
  	44  dyld                                0x0000000109ba92bf start_sim + 10
  	45  ???                                 0x00000002016aa52e 0x0 + 8613700910
  	46  ???                                 0x0000000000000003 0x0 + 3
   (ObjCRuntime.ObjCException)
     at ObjCRuntime.Runtime.ThrowException(IntPtr gchandle)
     at UIKit.UIApplication.UIApplicationMain(Int32 argc, String[] argv, IntPtr principalClassName, IntPtr delegateClassName)
     at UIKit.UIApplication.Main(String[] args, Type principalClass, Type delegateClass)
     at Microsoft.Maui.DeviceTests.Program.Main(String[] args) in /Users/jopepper/src/maui/src/Controls/tests/DeviceTests/Microsoft.Maui.TestUtils.DeviceTests.Runners.SourceGen/Microsoft.Maui.TestUtils.DeviceTests.Runners.SourceGen.CodeBehindGenerator/TestRunner.iOS.sg.cs:line 18
```